data/api/globals.lua

```diff
@@ -0,0 +1,22 @@
+--- @meta
+
+--- The command line arguments given to lite.
+--- @type table<integer, string>
+ARGS = {}
+
+--- The current operating system. The value can be:
+--- "Windows", "Mac OS X", "Linux", "iOS", "Android"
+--- @type string
```

Comment:
according to [this page](https://emmylua.github.io/annotations/string_literal_type.html), you can document string literals ith `@type string | "'Windows'" | "'Mac OS X'" | "'Linux'" | "'iOS'" | "'Android'"`

Reply:
Used that syntax and the @alias in all applicable params and returns, thank you!